core/src/epicli/data/common/ansible/playbooks/roles/node_exporter/tasks/main.yml

```
@@ -1,83 +1,113 @@
-    group: "node_exporter"
-    mode: 0750
-    state: directory
-  with_items:
-  - /opt/node_exporter
+    - name: Create /opt/node_exporter directories
+      become: true
+      file:
+        path: "{{ item }}"
+        recurse: true
+        owner: root
+        group: "node_exporter"
+        mode: 0750
+        state: directory
+      with_items:
+      - /opt/node_exporter
 
-- name: Unpack node_exporter binary
-  become: true
-  unarchive:
-    remote_src: true
-    src: "{{ download_directory }}/{{ exporter_file_name }}"
-    dest: "/opt/node_exporter"
-    creates: "/opt/node_exporter/node_exporter"
-    extra_opts: [--strip-components=1]
-    mode: 0755
-    owner: root
-    group: node_exporter
-  check_mode: false
-  notify:
-    - restart prometheus node exporter service
+    - name: Unpack node_exporter binary
+      become: true
+      unarchive:
+        remote_src: true
+        src: "{{ download_directory }}/{{ exporter_file_name }}"
+        dest: "/opt/node_exporter"
+        creates: "/opt/node_exporter/node_exporter"
+        extra_opts: [--strip-components=1]
+        mode: 0755
+        owner: root
+        group: node_exporter
+      check_mode: false
+      notify:
+        - restart prometheus node exporter service
 
-- name: Install node_exporter service to systemd
-  template:
-    src: prometheus-node-exporter.service.j2
-    dest: "/etc/systemd/system/{{ exporter.service.name }}.service"
-    owner: root
-    group: root
-    mode: 0644
+    - name: Install node_exporter service to systemd
+      template:
+        src: prometheus-node-exporter.service.j2
+        dest: "/etc/systemd/system/{{ exporter.service.name }}.service"
+        owner: root
+        group: root
+        mode: 0644
 
-- name: Configure systemd to use node_exporter service
-  systemd:
-    daemon_reload: true
-    enabled: true
-    name: "{{ exporter.service.name }}.service"
+    - name: Configure systemd to use node_exporter service
+      systemd:
+        daemon_reload: true
+        enabled: true
+        name: "{{ exporter.service.name }}.service"
 
-- name: Start exporter
-  service:
-    name: "{{ exporter.service.name }}"
-    state: started
+    - name: Start exporter
+      service:
+        name: "{{ exporter.service.name }}"
+        state: started
 
-- name: Copy file_sd_config to prometheus hosts
-  template:
-    dest: "{{ specification.config_for_prometheus.prometheus_config_dir }}/file_sd/node-{{ inventory_hostname }}.yml"
-    src: file_sd_config.yml.j2
-    owner: root
-    group: root
-    mode: 0644
-  delegate_to: "{{ item }}"
-  with_inventory_hostnames:
-    - prometheus
-  notify: restart prometheus on remote servers
+    - name: Copy file_sd_config to prometheus hosts
+      template:
+        dest: "{{ specification.config_for_prometheus.prometheus_config_dir }}/file_sd/node-{{ inventory_hostname }}.yml"
+        src: file_sd_config.yml.j2
+        owner: root
+        group: root
+        mode: 0644
```

How about going human readable? :thinking: 
```suggestion
        mode: u=rw,go=r
```